drop-transfer/src/service.rs

```diff
@@ -82,6 +86,8 @@ impl Service {
                 config,
                 auth: auth.clone(),
                 storage,
+                #[cfg(unix)]
+                fdresolv,
```

Comment:
Also, can we maybe expose this function just for UNIX? We need to make our APIs as hard to misuse as possible, so if Windows wouldn't even have a chance to see it in any way

Reply:
Do you mean moving it to a separate function? It would be more prone to misuse then because the callback can't be dynamically switched on down the row. It must be present from the beginning.
On the ffi layer, it's a different story. There we must somehow not bother apps with this callback except Android so as I see it is beneficial to instruct only the Android team to call this additional function